core/src/main/java/org/infinispan/util/logging/events/Messages.java

```diff
@@ -29,8 +29,8 @@
    @Message(value = "Started local rebalance", id = 100002)
    String rebalanceStarted();
 
-   @Message(value = "Finished local rebalance", id = 100003)
-   String rebalanceCompleted();
+   @Message(value = "Finished local rebalance phase", id = 100003)
+   String rebalancePhaseConfirmed();
```

Comment:
It's weird that none of these messages have parameters... I'd change it to "Node %s finished receiving state for rebalance %d" (with CacheTopologyControlCommand.rebalanceId), and the one before to "Started rebalance %d" (it seems to be used for the REBALANCE_START broadcast, so it's not local).